rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -283,11 +284,15 @@ def test_thegraph_balances_vested_arbitrum_one(
         inquirer: 'Inquirer',  # pylint: disable=unused-argument
 ) -> None:
     """Check that balances of GRT currently vested are properly detected."""
-    expected_grt_balances = [FVal('166666.67'), FVal('300000')]
+    expected_grt_balance = FVal('246914.83683300448')
     # decode the delegation transfer event which has the vested contract address as delegator_l2
```

Comment:
I suspect some of the errors in this file come from not making a proper abstraction on the logic